tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -72,6 +82,8 @@ PyMethodDef variable_methods[] = {
   {"contiguous", (PyCFunction)THPVariable_contiguous, METH_NOARGS, NULL},
   {"detach", (PyCFunction)THPVariable_detach, METH_NOARGS, NULL},
   {"detach_", (PyCFunction)THPVariable_detach_, METH_NOARGS, NULL},
+  {"nelement", (PyCFunction)THPVariable_numel, METH_NOARGS, NULL},
```

Comment:
this is also aliased as `numel`

Reply:
Right, ATen already has that so it works automagically.

Reply:
Right, I didn't notice that it's autogenerated in `py_method_defs`